### DISCUSSION TITLE
Is there a size limit for the extension to work?

### DISCUSSION THREAD

**Discussion by bwshn** [James-Yu/LaTeX-Workshop#4456] [Q&A] [answered]
Hi, recently I have reached 1.5 MB size in my main .tex file, and it causes "extension host terminated unexpectedly" thing. I assume it is caused by the size of the file since it does not happen when I don't launch the main file, and try to do with a lighter .tex file. I tried deleting the extension and related files.
Is there anything I can do other than splitting my .tex file? How can I troubleshoot this? Is it due to the limitations of VS Code itself?
Thanks in advance.

**Comment by James-Yu** [comment, marked as answer]
Very unlikely the extension would work with such a large file. The only way is to split it into multiple files.
